components/NavButtons.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { useTranslations } from 'next-intl'
+
+type Props = {
+  activeQuestion: number
+  connected: boolean
+  lastQuestion: boolean
```

Comment:
We should probably rename this to something like `isLastQuestion` so it's not confused for an action